index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width"/>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <div class="headerelement headerelement1">Welcome</div>
+        <div class="headerelement headerelement2"><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/nav-menu1.png" height="25" width="25" alt="navimg"/></div>
```

Comment:
- [ ] The hamburger/menu button should be enclosed between the `nav` tag. 
- [ ] the path should start `./` currently no image is being rendered on the online version of your project.

Reply:
Done, thank you

Reply:
Well reolved